cfg/mcuconf.h

```diff
@@ -51,13 +51,13 @@
  */
 
 #define HT32_GPT_USE_BFTM0                  TRUE
-#define HT32_GPT_BFTM0_IRQ_PRIORITY         4
+#define HT32_GPT_BFTM0_IRQ_PRIORITY         5
 
 #define HT32_SERIAL_USE_USART0              FALSE
 #define HT32_SERIAL_USE_USART1              TRUE
 #define HT32_USART1_IRQ_PRIORITY            3
 
-#define HT32_GPT_USE_BFTM1                  TRUE
+#define HT32_GPT_USE_BFTM1                  FALSE
```

Comment:
I am using `BFTM0` only, so disabling this.

Reply:
I think this is some previously leftover bug as well. So if you look at the [ARM documentation](https://developer.arm.com/documentation/dui0662/b/Cortex-M0--Peripherals/Nested-Vectored-Interrupt-Controller/Interrupt-Priority-Registers) for Cortex-M0+ which is what the HT32F523x2 is.
You can see the interrupt priority is only 2 bits. Thus only 0-3 are actually valid values. This value is shifted to the highest two bits of each byte, you can see this implemented [here](https://github.com/ChibiOS/ChibiOS/blob/338be05bb30e493b944187daada4c4efcc929fb8/os/hal/ports/common/ARMCMx/nvic.h#L69).

So we should uniform the priority to between 0-3. since 5 is actually going to be 1. Which I would assume is not what you want.

Reply:
updated

Reply:
ack